test/pylib/cpp/common_cpp_conftest.py

```diff
@@ -91,30 +110,8 @@ def collect_items(file_path: PosixPath, parent: Collector, facade: CppTestFacade
     if len(custom_args) > 1:
         return CppFile.from_parent(parent=parent, path=file_path, arguments=args, parameters=custom_args,
                                    no_parallel_run=no_parallel_run, modes=modes, disabled_tests=disabled_tests,
-                                   run_id=run_id, facade=facade, project_root=project_root)
+                                   run_id=run_id, facade=facade, env=test_env)
     else:
         args.extend(custom_args)
         return CppFile.from_parent(parent=parent, path=file_path, arguments=args, no_parallel_run=no_parallel_run,
-                                   modes=modes, disabled_tests=disabled_tests, run_id=run_id, facade=facade, project_root=project_root)
-
-
-@cache
-def get_combined_tests():
```

Comment:
nit: It would be nice if this method is moved to its location in boost_facade.py by a separate patch, then changed